test/hydro/nusselt_number_test.py

```diff
@@ -0,0 +1,83 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import assert_equal, Quantity, units, errors
+from symplyphysics.laws.hydro import nusselt_number
+
+# Example from: https://www.omnicalculator.com/physics/nusselt-number
+# Conditions:
+# Forced convection, moderate speed cross-flow of air over a cylinder.
+# Heat transfer coefficient equals 200 W / m*m*K (source https://www.engineersedge.com),
+# thermal conductivity for air equals 0.022 W / m*K, characteristic length equals 1 m.
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    heat_transfer_coefficient = Quantity(
+        200 * units.watt / (units.meter**2 * units.kelvin))
+    characteristic_length = Quantity(1 * units.meter)
+    thermal_conductivity = Quantity(
+        0.022 * units.watt / (units.meter * units.kelvin))
+    Args = namedtuple("Args", ["heat_transfer_coefficient",
+                      "characteristic_length", "thermal_conductivity"])
+    return Args(
+        heat_transfer_coefficient=heat_transfer_coefficient,
+        characteristic_length=characteristic_length,
+        thermal_conductivity=thermal_conductivity,
+    )
+
+
+def test_basic_nusselt_number(test_args):
```

Comment:
We have updated type checking. Sync with master and add types to your test. Like this: https://github.com/blackyblack/symplyphysics/blob/master/test/chemistry/atomic_number_density_from_material_density_atomic_weight_test.py